output/socket/outputsocket_test.go

```diff
@@ -0,0 +1,63 @@
+package socket
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stvp/go-udp-testing"
+	"github.com/tsaikd/gogstash/config"
+	"github.com/tsaikd/gogstash/config/logevent"
+	"math/rand"
+)
+
+var (
+	logger = config.Logger
+)
+
+func init() {
+	logger.Level = logrus.DebugLevel
+	config.RegistOutputHandler(ModuleName, InitHandler)
+}
+
+func Test_output_report_module(t *testing.T) {
+	assert := assert.New(t)
+	assert.NotNil(assert)
+	require := require.New(t)
+	require.NotNil(require)
+
+	udp.SetAddr(":9876")
```

Comment:
`undefined: udp` ??

Reply:
What do you mean?  What's `undefined`?

Reply:
I do `go test` and get an error message.
Do you forget commit some test code?

Reply:
Hm, that's very bizare...  It passes just fine on my mac, and the Travis-CI also passed the build.
This is what I see:
```
>> gogstash (outputsocket)$ cd output/socket/
>> socket (outputsocket)$ go test
PASS
ok  	github.com/tsaikd/gogstash/output/socket	7.349s
```

Reply:
Sorry, it's my fault. My atom removed 'github.com/stvp/go-udp-testing' when I save file...T_T

Reply:
Phew... false alarm... I started doubting my abilities there for a moment. 😜 
Glad to hear you got it sorted out.